cmds/ed/command.go

```diff
@@ -41,7 +42,8 @@ func Command(f Editor, c string, startLine, endLine int) error {
 	case 'r':
 		fname := strings.TrimLeft(a, " \t")
 		debug("read %v @ %v, %v", f, startLine, endLine)
-		r, err := os.Open(fname)
+		var r io.Reader
+		r, err = os.Open(fname)
 		debug("%v: r is %v, err %v", fname, r, err)
 		if err == nil {
 			_, err = f.Read(r, startLine, endLine)
```

Comment:
So this `err` is never checked because it immediately goes out of scope.

Reply:
ed is kinda crap. sorry about that.